bpf/init.sh

```diff
@@ -323,7 +321,7 @@ elif [ "$MODE" = "lb" ]; then
 	else
 		echo 1 > /proc/sys/net/ipv6/conf/all/forwarding
 
-		CALLS_MAP="cilium_calls_lb_${ID}"
+		CALLS_MAP="cilium_calls_lb"
```

Comment:
Should this be `CALLS_MAP="cilium_calls_lb_${ID_HOST}"` or ID_WORLD?

Reply:
No, LB is always a single program, this was a copy&paste error but it didn't have any negative effect.